bigchaindb/commands/utils.py

```diff
@@ -95,6 +96,34 @@ def start(parser, argv, scope):
     return func(args)
 
 
+def mongodb_host(host):
```

Comment:
Mmm yeah, and I guess we'd be importing `pymongo` anyway since the command parser needs it.

Maybe in the future we can have backend-specific commands (and some way to hook them up to the main command parser).